Proof of Concept/Spark_POC.ipynb

```diff
@@ -0,0 +1,471 @@
+{
```

Comment:
<p>Who did the fine tuning? You? Very nice if so.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/BU-Spark/ml-nlgma-body-cam/pull/2/discussion/#comment-1392925786'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>